cmd/main.go

```diff
@@ -116,9 +116,9 @@ func Main(args []string) error {
 	probe.SetAppInfo("Commit", ShortCommitID)
 
 	// Fetch terminal size, if not available, automatically
-	// set globalQuiet to true.
+	// set globalQuiet to true on non-window.
 	if w, h, e := term.GetSize(int(os.Stdin.Fd())); e != nil {
-		globalQuiet = true
+		globalQuiet = runtime.GOOS != "windows"
```

Comment:
@klauspost https://cs.opensource.google/go/x/term/+/refs/tags/v0.13.0:term_windows.go, it looks like the term package we are using supports Windows; I am not sure what is going on

Reply:
May be @klauspost has one of those special windows versions :-)

Reply:
@vadmeste I don't know. It fails on mine.